src/Turtle/Prelude.hs

```diff
@@ -367,61 +374,61 @@ stream p s = do
     inhandle hOut
 
 -- | Print to @stdout@
-echo :: Text -> IO ()
-echo = Text.putStrLn
+echo :: MonadIO io => Text -> io ()
+echo t = liftIO (Text.putStrLn t)
```

Comment:
I looked up parameter names in corresponding libraries. `text` calls the parameter `t` in the definition of `Text.putStrLn`. However, I too think that it's not as descriptive as it should be. I'll change that in a minute.